src/gui/qgsmaptoolcapture.cpp

```diff
@@ -30,6 +30,8 @@
 #include "qgsvectorlayer.h"
 #include "qgsvertexmarker.h"
 #include "qgssettings.h"
+#include "qgsapplication.h"
```

Comment:
this line in not present in master! is it necessary? seems not related with the added line in this file

Reply:
```suggestion
```

Reply:
Thanks it was the result of a merge conflict